mRemoteNG/mRemoteNG
# Pre-build event powershell code update #2524

## Description
For detailed information, please refer to issue #2524.

## Motivation and Context
Addressing and resolving identified issues.

## How Has This Been Tested?
- Open Visual Studio.
- Load the project from a location with or without spaces in the path.
- Build or rebuild the project.

Before the code update/pr, there is an issue with spaces in the project location path (for more information, see #2524).

## Screenshots (if appropriate):
Screenshot of new console output:
![image](https://github.com/mRemoteNG/mRemoteNG/assets/145561288/cc3d2cf0-3ab4-47f4-b391-fe9013fcef7d)

## Types of changes

- [ ] Changed feature (non-breaking change which changes functionality)
- [ ] Changed feature (**breaking** change which changes functionality)
- [ ] Updated translation

## Checklist:


- [x] All Tests within VisualStudio are passing
